docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -22,12 +22,13 @@ private Docker registry or repository.
 
 ## Log in to Docker
 
+On your laptop, you must authenticate with a registry in order to pull a private image:
```

Comment:
private registry is different from private image ...

Reply:
you must authenticate with a private registry in order to pull a image from it:
I guess it may be proper...